drivers/hmis/app/graphql/types/hmis_schema/external_form_submission.rb

```diff
@@ -20,10 +20,19 @@ class HmisSchema::ExternalFormSubmission < Types::BaseObject
     field :status, HmisSchema::Enums::ExternalFormSubmissionStatus, null: false
     field :notes, String, null: true
     field :definition, Forms::FormDefinition, null: false
-    custom_data_elements_field
+    field :values, GraphQL::Types::JSON, null: true
+    custom_data_elements_field # Resolve for backwards compatibility. Can remove in a future release
 
     def definition
       load_ar_association(object, :definition)
     end
+
+    def values
+      # Unlike CustomAssessments, which display on the frontend using the created CDEs once they've been submitted,
+      # ExternalFormSubmissions always display using the values from raw_data. Example: If the form processor created
+      # a client, and then a user manually updates the client's name in HMIS, the form review UI should still display
+      # the client name that was originally submitted with the form.
+      object.raw_data
```

Comment:
We probably also want to exclude the "extraneous keys" here. Maybe it makes sense to have a function on the `HmisExternalApis::ExternalForms::FormSubmission`, so that list of keys also lives closer to home:

```
# keys that are added to raw_values upon submission, but are not part of the form values
EXTRANEOUS_KEYS = ['captcha_score', 'form_definition_id', 'form_content_digest'].freeze

def form_values
   raw_data.reject { |key, _| EXTRANEOUS_KEYS.include?(key) }
end
```